src/jit/emit.cpp

```diff
@@ -5533,6 +5562,111 @@ void emitter::emitOutputDataSec(dataSecDsc* sec, BYTE* dst)
     }
 }
 
+#ifdef DEBUG
+void emitter::emitDispDataSec(dataSecDsc* section)
```

Comment:
Nice! Might I ask for a basic function header?

Reply:
Sure, will add one.